### PR TITLE
[6] Develop a Product Page Using HTML

### PR DESCRIPTION
**Pull request by john84edeh**
## Task: `[6] Develop a Product Page Using HTML`
- `I updated the e-commerce site following the instruction giving and uploaded all images i used to make the pull request
Bride's-made-gown
Pink-lace-rossy-buba
WhatsApp Image 2020-07-22 at 16 05 53
WhatsApp Image 2020-07-22 at 16 05 54 (1)
WhatsApp Image 2020-07-22 at 16 05 54 (2)
WhatsApp Image 2020-07-22 at 16 05 54
WhatsApp Image 2020-07-22 at 16 05 55

`

### CODE REVIEW

**Comment by ToluAkin** [comment]
@john84edeh You are deleting your files. Kindly update.

File: submissions/edehJohn/html/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/edehJohn/html/index.html:1]: There suppose to be a ordered list of similar products. You have not added it.

File: submissions/edehJohn/html/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    <h4>Description</h4>
+                    <article>
+                        <p>
+                            <a href="#"> other styles and Accessories clique</a>
+                        </p>
+                    </article>
+                </ul>
+            </ul>
+            
+        </nav>
```
[the remark this answers is not in the document]
Comment (john84edeh) [reply submissions/edehJohn/html/index.html:48]: Thank u , have sort it out

File: submissions/edehJohn/html/index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <ul>
+                <li><a href="#">Login</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Move to chart</a></li>
+            </ul>
+            <h2><strong>Our Products and Services</strong></h2>
+            <ul>
+                <a href="#">Fashion & Design</a>
+                <ul>
+                    <img src="./img/Bride's-made-gown.jpeg" alt="My Gown"/>
```
[the remark this answers is not in the document]
Comment (john84edeh) [reply submissions/edehJohn/html/index.html:19]: the images are there and they are displaying on the explorer i was told to review it to just two as requested which i did.

**Pull request closed without merging**